Description: remove random from cabal file
 Remove random from cabal file, as this package is split out of ghc 7.4,
 and is not actually used by haxml.
Author: Joey Hess <joeyh@debian.org>

--- haxml-1.22.5.orig/HaXml.cabal
+++ haxml-1.22.5/HaXml.cabal
@@ -67,7 +67,7 @@ library
   hs-source-dirs: src
   build-depends: polyparse >= 1.5, filepath
   if flag(splitBase)
-    build-depends: base >= 3 && < 6, pretty, random, containers
+    build-depends: base >= 3 && < 6, pretty, containers
   else
     build-depends: base < 3
   if flag(bytestringInBase)
